backend/internal/auth/login_timing_test.go

```
@@ -0,0 +1,132 @@
+	}()
+
+	ticker := time.NewTicker(time.Millisecond)
+	defer ticker.Stop()
+	timeout := time.NewTimer(2 * time.Second)
+	defer timeout.Stop()
+	for len(gate.admitted) < 2 {
+		select {
+		case err := <-loginDone:
+			cancel()
+			close(release)
+			_ = <-blockerDone
```

golangci-lint (2.13.2)

[error] S1005: unnecessary assignment to the blank identifier

(staticcheck)